src/applications/mhv/secure-messaging/components/Dashboard/WelcomeMessage.jsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { mhvUrl } from '@department-of-veterans-affairs/platform-site-wide/utilities';
+import { isAuthenticatedWithSSOe } from '~/platform/user/authentication/selectors';
 import FeedbackEmail from '../shared/FeedbackEmail';
 
 const WelcomeMessage = () => {
+  const fullState = useSelector(state => state);
   return (
     <div className="welcome-message">
       <h2>What to know as you try out this tool</h2>
       <p>
         We’re giving the trusted My HealtheVet secure messaging tool a new home
-        here on VA.gov. You can use this tool to communicate securely with your
-        care team online-just like you can today on the My HealtheVet website.
+        on VA.gov. And we need your feedback to help us keep making this tool
+        better for you and all Veterans.
       </p>
       <p>
-        We need your feedback to help us keep making this tool better for you
-        and all Veterans.
+        Email your feedback and questions to us at <FeedbackEmail />.
       </p>
       <p>
-        Email us at <FeedbackEmail /> to tell us what you think. We can also
-        answer questions about how to use the tool.
+        <strong>Note:</strong> You still have access to the previous version of
+        secure messaging. You can go back to that version at any time.{' '}
+        <a
+          href={mhvUrl(isAuthenticatedWithSSOe(fullState), 'secure-messaging')}
+          target="_blank"
```

Comment:
Is the goal to make a new window for this link or just redirect the user's window to the link?